int32[4] box
string class
float32 score